managed_vms/django_tutorial/README.md

```diff
@@ -0,0 +1,103 @@
+# Getting started with Django on Google Cloud Platform
+
+This repository is an example of how to run a [Django](https://www.djangoproject.com/) 
+app on Google Managed VMs. It uses the [Writing your first Django app](https://docs.djangoproject.com/en/1.9/intro/tutorial01/) as the example app to deploy.
+
+
+## Setup the database
+
+This tutorial assumes you are setting up Django using a SQL database, which is the easiest way to run Django. If you have an existing SQL database running, you can use that, but if not, these are the instructions for creating a managed MySQL instance using CloudSQL.
+
+
+* Create a [CloudSQL instance](https://console.cloud.google.com/project/_/sql/create)
```

Comment:
These instructions go out of date so quickly. Do you think it would be better to provide the gcloud commands instead? You could mention that it's possible through the console.